src/index.js

```diff
@@ -0,0 +1,9 @@
+import ReactDOM from 'react-dom';
```

Comment:
- [ ] It would be great if you could fix the props warnings
![Screenshot from 2022-01-05 15-58-21](https://user-images.githubusercontent.com/62705947/148239270-2003798e-7c36-4efd-8867-c121e568c10f.png)

Reply:
Solved